.changelog/unreleased/improvements/ibc/1618-get-frozen-height.md

```diff
@@ -0,0 +1,2 @@
+- Add the `frozen_height()` method to the `ClientState` trait
```

Comment:
Nit: this PR contains some breaking changes although the changelog entry is in the `improvements` section.

Reply:
https://github.com/informalsystems/ibc-rs/pull/1619/commits/9e975bd030e404d7c128f46b64eeee4b2e175108